fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1411,13 +1411,13 @@ ACTOR Future<GetReadVersionReply> getLiveCommittedVersion(ProxyCommitData* commi
 	rep.version = commitData->committedVersion.get();
 	rep.locked = commitData->locked;
 	rep.metadataVersion = commitData->metadataVersion;
-	rep.recentRequests = commitData->stats.getRecentRequests();
 
 	for (auto v : versions) {
 		if(v.version > rep.version) {
 			rep = v;
 		}
 	}
+	rep.recentRequests = commitData->stats.getRecentRequests();
```

Comment:
It seems like the change is moving this line to here after the for loop. What’s the problem it fixes?

Reply:
It seems that I cannot configure the reviewer to review the PR. Can you help add Evan into this since I have synced with him about this offline.

Reply:
Have updated the PR description, thanks.

Reply:
Oh I see the problem.